packages/devcmd/src/process/ProcessExecutor.ts

```diff
@@ -29,9 +33,17 @@ class SafeConsoleLike implements ConsoleLike {
   }
 }
 
+/** 
+ * A class abstracting the execution of tasks in separate child processes
+ */
 export class ProcessExecutor {
   private readonly consoleLike: ConsoleLike;
 
+  /**
+   * Constructor
+   * 
+   * @param {ConsoleLike} consoleLike A {@link ConsoleLike} for logging purposes
```

Comment:
the type annotations in JSDoc are redundant, because we use TS and have the information there.

tbh i'm not sure how other consumers of JSDoc handle this in TS files, but VS Code doesn't even show these type annotations. I think we can just remove them -- they're just likely to become outdated if we ever change the code itself.

Reply:
Removing the type annotations